packages/app/src/components/NicksPetList.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <div class="box-content bg-blue-500 m-2 p-6">
```

Comment:
Just a personal preference. With the max-w the blue container does not span the whole Screen. Looks weird on an ultrawide.

```suggestion
  <div class="box-content mx-auto max-w-screen-xl bg-blue-500 m-2 p-6">
```